lighthouse-core/gather/gatherers/http-redirect.js

```diff
@@ -36,39 +36,20 @@ class HTTPRedirect extends Gatherer {
   }
 
   afterPass(options) {
-    // Reset the options.
-    options.url = this._preRedirectURL;
-
-    // Allow override for faster testing.
-    const timeout = options._testTimeout || 10000;
-
-    const securityPromise = options.driver.getSecurityState()
-      .then(state => {
-        return {
-          value: state.schemeIsCryptographic
-        };
+    let isRedirect = false;
+
+    if(options.driver._httpsArr.length !== 0) {
+      const httpsExpectURL = options.url.replace(/^http/, 'https');
+      options.driver._httpsArr.forEach( (httpsUrl) => {
+        if(httpsUrl.url === httpsExpectURL) {
```

Comment:
I formed this after `http://m.uber.com` fails to resolve to https on the prev approach. With `'/'`, we are assuming the root route is https if it's in the arr. Anything I need to change here?